Modules/WebSocket/Sources/WebSocket/Buffer.swift

```diff
@@ -0,0 +1,73 @@
+import Core
+
+#if os(Linux)
+    import Glibc
+#else
+    import Security
+#endif
+
+extension Buffer {
```

Comment:
I think these should be added to `Buffer`. We should probably wait for #162 before merging this one and then move these or some of these there.